Add documentation for enabling tracing (#4691)
EnMasseProject/enmasse: documentation/modules/con-iot-tracing.adoc

// Module included in the following assemblies:
//
// assembly-iot-service-admin-guide.adoc

[id='con-iot-tracing-{context}']
= IoT tracing

{ProductName} allows to enable application level tracing in the IoT infrastructure

> using Jaeger tracing. This is a feature to allow service admins gain insights on the

Reviewer: ```suggestion
using Jaeger. This feature allows service administrators to gain insight into the
```

inner workings of the IoT services in order to analyze performance and issues in the system.

> By default, tracing support is not enabled and needs to be activated manually. Learn how to link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#proc-iot-tracing-messaging-iot-iot-iot[Configuring tracing].

Reviewer: ```suggestion
By default, tracing support is not enabled and needs to be activated manually. For more information, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#proc-iot-tracing-messaging-iot-iot-iot[Configuring tracing].
```
